templates/base_index.html

```diff
@@ -106,7 +106,7 @@ <h2 class="p-heading--1 u-no-margin--bottom">Modern enterprise open&nbsp;source<
             </p>
           </div>
           <p>
-            <a class="p-button is-dark" href="/pro">Get Ubuntu Pro</a>
+            <a class="p-button p-button--positive" href="/pro">Get Ubuntu Pro</a>
```

Comment:
```suggestion
            <a class="p-button--positive" href="/pro">Get Ubuntu Pro</a>
```
Your approach follows BEM but we typically just use the active class